pandas/core/generic.py

```diff
@@ -7515,29 +7515,91 @@ def _check_percentile(self, q):
         return q
 
     _shared_docs['pct_change'] = """
-        Percent change over given number of periods.
+        Percentage change between the current and previous element.
+
+        This is useful in comparing the percentage of change in a time series
+        of elements.
 
         Parameters
         ----------
         periods : int, default 1
-            Periods to shift for forming percent change
+            Periods to shift for forming percent change.
         fill_method : str, default 'pad'
-            How to handle NAs before computing percent changes
+            How to handle NAs before computing percent changes.
         limit : int, default None
-            The number of consecutive NAs to fill before stopping
+            The number of consecutive NAs to fill before stopping.
         freq : DateOffset, timedelta, or offset alias string, optional
-            Increment to use from time series API (e.g. 'M' or BDay())
+            Increment to use from time series API (e.g. 'M' or BDay()).
+        **kwargs : mapping, optional
+            Additional keyword arguments are passed into
+            ``DataFrame.shift``/``Series.shift``.
 
         Returns
         -------
-        chg : %(klass)s
+        chg : Series or DataFrame, same type as the calling object
 
         Notes
         -----
-
         By default, the percentage change is calculated along the stat
```

Comment:
I don't think most users will understand `stat` access. Best to remove this and only focus on Series / DataFrame.